src/main.py

```diff
@@ -79,6 +79,9 @@ def main(module_name: OracleModule):
     logger.info({'msg': 'Initialize keys api client.'})
     kac = KeysAPIClientModule(variables.KEYS_API_URI, web3)
 
+    logger.info({'msg': 'Initialize IPFS providers.'})
+    ipfs = MultiIPFSProvider(ipfs_providers())
```

Comment:
pass here or somewhere `IPFS_RETRIES` env var

Reply:
it should be configurable by user as for other providers

Reply:
Done